ansible/roles/sonic/tasks/vsonic.yml

```diff
@@ -64,3 +64,8 @@
   when: hostname in configuration
   notify:
     - Restart BGP service
+
+- name: Load PTF image
+  block:
+    - copy: src="docker-ptf.tar" dest="docker-ptf.tar"
+    - shell: docker load -i docker-ptf.tar || true
```

Comment:
Cleanup the docker-ptf.tar file from local and remote after "docker load -i" is done?

Reply:
Done